packages/camera/camera_android_camerax/example/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Demonstrates how to use the camera_android_camerax plugin.
 publish_to: 'none'
 
 environment:
-  sdk: '>=2.17.0 <3.0.0'
+  sdk: ">=2.17.0 <4.0.0"
```

Comment:
The changes to all of the packages was auto-generated (by a slightly-locally-modified version of `update-min-sdk`) which is why the quoting changed in packages that were using `'` instead of `"`.